tesseract_environment/test/tesseract_environment_unit.cpp

```diff
@@ -475,20 +475,21 @@ void runApplyCommandsTest()
       EXPECT_TRUE(std::find(joint_names.begin(), joint_names.end(), joint_name1) != joint_names.end());
     }
     {
-      Commands commands{ std::make_shared<AddCommand>(link_2, nullptr) };
+      Commands commands{ std::make_shared<AddCommand>(link_2) };
       EXPECT_TRUE(env->applyCommands(commands));
       std::vector<std::string> link_names = env->getLinkNames();
       std::vector<std::string> joint_names = env->getJointNames();
       EXPECT_TRUE(std::find(link_names.begin(), link_names.end(), link_name2) != link_names.end());
       EXPECT_TRUE(std::find(joint_names.begin(), joint_names.end(), "joint_" + link_name2) != joint_names.end());
     }
     {
-      Commands commands{ std::make_shared<AddCommand>(nullptr, joint_1) };
-      EXPECT_FALSE(env->applyCommands(commands));
+      EXPECT_DEATH(Commands commands{ std::make_shared<AddCommand>(nullptr, joint_1) }, "");
```

Comment:
Should we fix this? This seems less than ideal.

Reply:
I will fix it such that an exception is thrown in the constructor of the AddCommand class.

Reply:
I switched command to take const& which solved solves the issue.